packages/deployer/internal/initializable-ast-validator.js

```diff
@@ -3,22 +3,27 @@ const {
   findContractDependencies,
   findFunctions,
 } = require('@synthetixio/core-js/utils/ast/finders');
+const { contractIsModule } = require('../internal/contract-helper');
 const { capitalize } = require('@synthetixio/core-js/utils/misc/strings');
 
 class ModuleInitializableASTValidator {
-  constructor(asts) {
+  constructor(asts, isModuleChecker = contractIsModule) {
```

Comment:
This new parameter is used to be able to fake the contractIsModule in unit tests. Otherwise it will require the hre initialized